internal/cli/atlas/accesslists/delete.go

```diff
@@ -54,15 +54,17 @@ func DeleteBuilder() *cobra.Command {
 		Use:     "delete <entry>",
 		Aliases: []string{"rm"},
 		Short:   "Delete an IP access list from your project.",
+		Long:    "The command prompts you to confirm the operation when you run the command without the --force option.",
 		Args:    require.ExactArgs(1),
 		Annotations: map[string]string{
-			"entryDesc": "The IP address, CIDR address, or AWS security group ID of the access list entry to delete.",
+			"args":         "entry",
+			"requiredArgs": "entry",
```

Comment:
We should be able to auto-discover this now
```suggestion
```

Reply:
@gssbzn I can remove this, but by "should" do you mean that this is possible to auto-discover but hasn't been done yet? I'm starting an effort to backfill most content from the old docs and want to make sure it's ok to leave these out for all commands

Reply:
by should I mean let us know if not happening because we have a bug we should fix